rpc/wrtc_server.go

```diff
@@ -85,6 +131,7 @@ func newWebRTCServerWithInterceptorsAndUnknownStreamHandler(
 		streamInt:         streamInt,
 		unknownStreamDesc: unknownStreamDesc,
 	}
+	srv.counters.CallTicketsAvailable.Store(int32(DefaultWebRTCMaxGRPCCalls))
```

Comment:
Are you not using `len(callTickets)` in `Stats` (and relying on this atomic) because of worries about a data race? I didn't think `len(callTickets)` concurrently with `<-callTickets` or `callTickets<-` would be cause for a race, but maybe I am wrong.

Reply:
Forgot len/cap worked on channels -- changed.